common_definitions.jl

```diff
@@ -0,0 +1,23 @@
+using Pkg
+Pkg.activate(@__DIR__)
+
+using KernelAbstractions
+
+if GPU_PKG_NAME == "CUDA"
+using CUDA, CUDAKernels
+const GPUMOD = CUDA
+const GpuArray = CuArray
+const GpuBackend = CuDevice()
+elseif GPU_PKG_NAME == "AMDGPU"
+using AMDGPU, ROCKernels
```

Comment:
I'd rather not import multiple packages. It's unclear how CUDA.jl's overrides interact with AMDGPU/oneAPI (probably badly).

Reply:
Ah never mind, it's conditional.